lib/src/base-styles/_nx-page-header.scss

```diff
@@ -8,65 +8,74 @@
 @import '../scss-shared/nx-text-helpers';
 @import '../scss-shared/nx-container-helpers';
 
+$main-header-background: $nx-sonatype-blue;
+$main-header-color: $nx-off-white;
+$main-header-highlight: $nx-sonatype-orange;
+$main-header-height: 72px;
+$main-header-branding-height: 52px;
+$main-header-product-grid-rows: $main-header-branding-height / 2;
```

Comment:
Is this right, considering the 2px gap?  Also would it be better called `$main-header-product-grid-rows`?

Reply:
> Is this right, considering the 2px gap? Also would it be better called `$main-header-product-grid-rows`?

https://github.com/sonatype/sonatype-react-shared-components/pull/13/commits/ddbab71786239da9d3fc391fde51098a068df712